umbraco-cms/reference/configuration/loggingsettings.md

```diff
@@ -1,25 +1,32 @@
 ---
-versionFrom: 9.0.0
-versionTo: 10.0.0
-meta.Title: "Umbraco Logging Settings"
-meta.Description: "Information on the logging settings section"
+description: "Information on the logging settings section."
 ---
 
 # Logging settings
 
 The majority of logging related configuration has been moved to the Serilog configuration see [Serilog settings](serilog.md) for more information.
 
-This settings section allows you to configure the max log age for the internal audit log scrubbing. The default maximum age for the internal audit log is 24 hours, however if you want to change this duration, you can with the `"MaxLogAge"` key, like so:
-
+The following configuration is available in the Logging settings:
 
 ```json
 "Umbraco": {
   "CMS": {
     "Logging": {
-      "MaxLogAge": "2.00:00:00"
+      "MaxLogAge": "2.00:00:00",
+      "Directory": "~/CustomLogFileLocation"
     }
   }
 }
 ```
 
-This will increase the maximum age of the entires in the audit log to 48 hours (2 days).
+## MaxLogAge
+
+This setting allows you to configure the maximum log age for the internal audit log scrubbing. The default maximum age for the internal audit log is 24 hours. Change the duration with the `"MaxLogAge"` key in the Logging settings.
```

Comment:
Since `"MaxLogAge"` already has a code tag, does it need "" as well? Below Directory is mentioned only as `Directory`